src/test/java/org/leeu1911/JsonBoxTest.groovy

```diff
@@ -4,7 +4,7 @@ import spock.lang.Specification
 
 def class JsonBoxTest extends Specification {
     def setup() {
-        JsonBox.boxId = "json_box_java_test_22092019"
+        JsonBox.boxId = UUID.randomUUID().toString().replaceAll("-","_")
```

Comment:
I wanted to fixed the `boxId` to not pollute jsonbox with test data for every time test suite runs. Did you have a problem with your tests so you had to change this? If not, let's revert this change.